icu4c/source/i18n/msgfmt.cpp

```diff
@@ -1683,6 +1688,7 @@ Format* MessageFormat::createAppropriateFormat(UnicodeString& type, UnicodeStrin
     Format* fmt = NULL;
     int32_t typeID, styleID;
     DateFormat::EStyle date_style;
+    int32_t first_non_space;
```

Comment:
Despite what date_style looks like, we should use camelCase variable names in ICU, like in Java.

Reply:
Done